src/larvaworld/lib/reg/models.py

```diff
@@ -851,9 +853,13 @@ def autogenerate_confs(self):
         olf_pars2 = self.generate_configuration(self.dict.brain.m['olfactor'].mode['default'].args,
                                                 gain_dict={'CS': 150.0,
                                                            'UCS': 0.0})
+        olf_brian_osn = self.generate_configuration(self.dict.brain.m['olfactor'].mode['brian_osn'].args,
+                                                gain_dict={'Odor': 150.0})
```

Comment:
Is this the place where I would add the BrianOlfactor constructor args?
For example:
`response_key='OSN_rate', server_host='localhost', server_port=5795, remote_dt=100, remote_warmup=0,`